### PR TITLE
Fix multiple maps on same dashboard

### PR DESCRIPTION
**Pull request by niknbr**
### Description
Wrapped private variables and methods in a function.

Fix for [INVE-11315]

multiple_maps

[INVE-11315]: https://sirensolutions.atlassian.net/browse/INVE-11315

### CODE REVIEW

File: public/lib/dragAndDroplayercontrol/DndLayerControl.js [hunk 1]
```diff
@@ -21,415 +21,422 @@ import EsLayer from './../../vislib/vector_layer_types/EsLayer';
...
-    dndCurrentListOrder={_allLayers}
-    dndListOrderChange={dndListOrderChange}
-    dndLayerVisibilityChange={dndLayerVisibilityChange}
-    dndRemoveLayerFromControl={dndRemoveLayerFromControl}
-  >
-  </LayerControlDnd >, _dndListElement);
-}
+  function _updateLayerControl() {
+    render(<LayerControlDnd
+      key={new Date().getTime()}
```
Comment (szydan) [inline public/lib/dragAndDroplayercontrol/DndLayerControl.js]: In other places we use 
uuid.v1()
Might be also easier to stub in tests if needed

**Comment by fabiocorneti** [comment]
Can you double check that there aren't elements with the same ID in the dom when the two maps are enabled?

**Comment by niknbr** [comment]
> Can you double check that there aren't elements with the same ID in the dom when the two maps are enabled?

There are a few. Will update this PR.

**Comment by jccq** [comment]
Well spotted Fabio!! :)

>

**Comment by niknbr** [comment]
image

_Note: Printed ID instead of label_

Updated `id` and `data-test-subj`(@johnnylopes)  of the drag and drop layers to `etm-vis-{panel-id}-{layer-id}`

**Pull request merged**